tests/reducers/medication-reducers.test.js

```diff
@@ -161,6 +161,35 @@ describe('Medication Reducers', () => {
     });
   });
 
+  describe('UPDATE_FHIR_MEDICATION_ORDER', () => {
+    it('handles the UPDATE_FHIR_MEDICATION_ORDER action for DSTU2', () => {
```

Comment:
Thank you for writing extra tests!  This is great!

By way of coordinating concurrent PRs, I wanted to note that Josh is removing UPDATE_FHIR_MEDICATION_ORDER and exporting the createFhirResource function here - https://github.com/cds-hooks/sandbox/pull/97/files#diff-6f0d41d697f90ffa883fb741cdf80bd1

To align with that, would you mind refactoring these tests a bit to focus on testing the createFhirResource function instead of UPDATE_FHIR_MEDICATION_ORDER?  His PR [removes the STU3-focused test](https://github.com/cds-hooks/sandbox/pull/97/files#diff-75c0be09c7a308a6ee71c58bb8db6843) but this would help preserve some tests for that logic.

Reply:
I've updated the tests.